ner4opt/exceptions.py

```diff
@@ -0,0 +1,7 @@
+"""Exceptions."""
+
+
+class InvalidModelError(Exception):
```

Comment:
not important, just curious; what's the advantage of having this vs. raising InvalidValueError()

Reply:
Thanks for pointing. For now, it does look like an overkill, reverted back to ValueError.